.hound.yml

```diff
@@ -1,6 +1,9 @@
 coffeescript:
   enabled: true
   config_file: style/coffeescript/coffeelint.json
+eslint:
+  enabled: true
+  config_file: style/eslint/.eslintrc
```

Comment:
There already seems to be an ESLint configuration present here - https://github.com/thoughtbot/guides/blob/master/style/javascript/.eslintrc.json. Maybe that should be used instead?